src/test/scss/data/fozzie-snapshot.scss

```diff
@@ -0,0 +1,128 @@
+@use 'fozzie' as f;
+
+// This file should call every part of Fozzie.
```

Comment:
Do we need to add something to the readme about unit testing? I assume every time a new component or style is added we need to modify this file? Does the order of the includes here break the snapshots too?

Reply:
Happy to update the readme. The order would absolutely break the snapshot as we're changing the outputted CSS :)

Reply:
The idea was to get started with a big snapshot of everything. Unit tests could get a bit more granular but I'd be happy to do individual snapshots on parts of the library in theory too. I think for this ticket, 1 snapshot of everything is appropriate though

Reply:
happy to create additional tickets

Reply:
Sounds good 👍 I think granular would make it much easier to understand / add too in future.